runtime/common/metering.go

```diff
@@ -63,16 +63,96 @@ func BigIntByteLength(v *big.Int) int {
 
 func NewPlusBigIntMemoryUsage(a, b *big.Int) MemoryUsage {
 	return NewBigIntMemoryUsage(
+		// TODO:
```

Comment:
Is this missing text or should it just be a lonely TODO?

Reply:
Just a TODO marker which we can look for when we are merging the feature branch

Reply:
Added links in [a48f222](https://github.com/onflow/cadence/pull/1493/commits/a48f222ba3540f98d50946baac70d956319a257a)